api/v1/model_types.go

```diff
@@ -144,6 +148,33 @@ type Adapter struct {
 	URL string `json:"url"`
 }
 
+type LoadBalancing struct {
+	Strategy LoadBalancingStrategy `json:"strategy"`
+	CHWBL    CHWBL                 `json:"chwbl"`
```

Comment:
nit, prefer to use the full name instead of abbreviations that most people don't understand. It's fine if it's long.

Reply:
Changed it to `PrefixHash` since this is a user-facing API and thats probably easier to understand.